.travis.yml

```diff
@@ -108,19 +109,22 @@ matrix:
             - *linux_deps
 
 before_install:
+# force python3
+  - sudo rm -rf /usr/bin/python
+  - sudo ln -s /usr/bin/python3 /usr/bin/python
+  - sudo apt-get update
+  - sudo apt-get install -y python3 python3-pip python3-setuptools
+  - sudo python -m pip install scons
```

Comment:
I'd look at using the `--user` pip switch then appending `$HOME/.local/bin` to the `PATH` so you can avoid `sudo pip` and all its caveats :slightly_smiling_face:

```shell
python -m pip install --user scons
export PATH="$HOME/.local/bin:$PATH"
scons …
```

Reply:
Thanks! I have updated the code to use this method as it's cleaner.